src/main/java/com/squareup/kotlinpoet/FileSpec.kt

```diff
@@ -271,7 +272,15 @@ class FileSpec private constructor(builder: FileSpec.Builder) {
       this.indent = indent
     }
 
-    fun build() = FileSpec(this)
+    fun build(): FileSpec {
+      annotations.forEach { annotationSpec ->
```

Comment:
Please use a regular for loop wherever possible.

Reply:
6755c8d